src/main/views/contact-the-tribunal.njk

```diff
@@ -17,9 +17,14 @@
         {{ landing_text.p2 }} <br><br>
         {{ landing_text.p3 }}
       </p>
+
       {{ govukAccordion({
         id: "contact-options",
-        items: applicationsAccordionItems
+        items: applicationsAccordionItems,
```

Comment:
Does this change the UI at all? If so it may need to be wrapped in a feature flag

Reply:
I don't think so. Tried toggling the feature flag on the preview link and don't see any problems.